db/schema.rb

```diff
@@ -933,6 +933,8 @@
     t.string "notes"
     t.date "induction_start_date"
     t.date "induction_completion_date"
+    t.date "mentor_completion_date"
+    t.string "mentor_completion_reason"
```

Comment:
from previous conversations I was wondering if it would be preferable to store multiple reasons instead of one, as that might apply to participants?

Reply:
I don't believe we need to support multiple current reasons, NAthan has confirmed there will only ever be one and will let us know the order of precedent when we support the other reasons. I will confirm again whether there is any need to support historic reasons if there is a change but I am pretty sure that wasn't necessary.

Reply:
Confirmation from policy colleagues that papertrail is enough, we don't need to support any other history for the dates/reason